promql/test.go

```diff
@@ -461,7 +461,7 @@ func (t *Test) exec(tc testCommand) error {
 		}
 
 		// Check query returns same result in range mode,
```

Comment:
I think the `,` at the end could be removed as well.

Reply:
Sorry, I can't understand what you mean.

Reply:
I don't think the `,` at the end of `... result in range mode,` is necessary. The previous comment for this function was different so maybe this one was incomplete or edited a few times but this was left in unintentionally. cc @brian-brazil

Reply:
I think either way works

Reply:
Ok, I have deleted this line of comments.

Reply:
This comment should remain

Reply:
Ok, I am going to modify it now.